modules/canbus/proto/BUILD

```diff
@@ -5,6 +5,24 @@ load("//tools:python_rules.bzl", "py_proto_library")
 
 package(default_visibility = ["//visibility:public"])
 
+cc_proto_library(
```

Comment:
This BUILD file  was generated by running `scripts/proto_build_generator.py modules/canbus/proto/BUILD`